package.json

```diff
@@ -4,61 +4,24 @@
   "description": "",
   "main": "dist/main.js",
   "scripts": {
-    "build": "rm -rf dist && tsc",
-    "watch": "tsc -w & nodemon",
-    "start": "node dist/main.js",
-    "start:dev": "npm run build && npm run watch",
-    "test-ci": "npm test",
-    "test:watch": "npm test -- --watchAll",
-    "test": "jest \"\\.spec\\.ts\"",
-    "test-integrations": "jest \"\\.integration\\.ts\" --forceExit",
     "lint-check": "eslint --fix-dry-run \"src/**/*.ts\"",
     "lint-fix": "eslint --fix \"src/**/*.ts\""
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Pocket/shared-snowplow-consumer.git"
+    "url": "git+https://github.com/Pocket/account-delete-monitor.git"
   },
```

Comment:
Should this be the other way around - `shared-snowplow-consumer` added rather than taken out?

Reply:
yes it should, copy paste error 🙈

Reply:
✅